pytroll_collectors/segments.py

```diff
@@ -491,6 +535,23 @@ def _find_time_slot(self, time_obj):
 
         return str(time_obj)
 
+    def check_schedule_time(self, check_time, raw_hour, raw_minute):
+        # Check if raw time is inside configured interval
```

Comment:
A docstring should start and finish with """ (better than #)
Also you need a . at the end of your sentence :)